### ISSUE TITLE
Tooltips doesn't show when cursor is over a fully-qualified class name

### ISSUE THREAD

**Issue by ricpelo** [php-integrator/atom-tooltips#26]
With the following code excerpt:

```php
<?php

namespace app\assets;

use yii\web\AssetBundle;

class AppAsset extends AssetBundle
{
    public $basePath = '@webroot';
    public $baseUrl = '@web';
    public $css = [
        'css/site.css',
    ];
    ...
}
```

if I put the mouse cursor over the `AssetBundle` in the class definition, the tooltip shows and all works OK.

However, if I refactor like this:

```php
<?php

namespace app\assets;

class AppAsset extends \yii\web\AssetBundle
{
    public $basePath = '@webroot';
    public $baseUrl = '@web';
    public $css = [
        'css/site.css',
    ];
    ...
}
```

then the tooltip doesn't show when I put the mouse over the `\yii\web\AssetBundle`.

**Comment by ricpelo** [comment]
```
$ apm list
[...]
├── php-integrator-annotations@1.1.1
├── php-integrator-autocomplete-plus@1.2.0
├── php-integrator-base@2.1.9
├── php-integrator-call-tips@1.2.1
├── php-integrator-linter@1.3.0
├── php-integrator-navigation@1.1.1
├── php-integrator-refactoring@1.3.1
├── php-integrator-tooltips@1.1.3
[...]
```

**Comment by Gert-dev** [comment]
Nice spot! Seems like this was a regression in Atom 1.13 with the new syntax selectors. This should be fixed with the just-released 1.1.4 release.

**Comment by ricpelo** [comment]
Thank you very much for your superb job, @Gert-dev !! :)

**Comment by ricpelo** [comment]
Hi, @Gert-dev . In the following code excerpt:

```php
<?php

namespace app\controllers;

use Yii;
use app\models\Film;

class RentController extends \yii\web\Controller
{
    [...]
}
```

Now the tooltip shows OK when I put the mouse over `\yii\web\Controller`, but it doesn't show when I put it over `app\models\Film` in the `use` line. However, if I change that line and put a `\` prefix (`use \app\models\Film`) then it works OK. Is it intentional?

**Comment by Gert-dev** [comment]
It does seem like both should be working, I'll investigate it.

**Comment by Gert-dev** [comment]
It seems that it broke with the Atom 1.13 updates. I fixed it in 1.1.5. Let me know if you find any other issues!
